SimCalorimetry/HcalSimProducers/python/hcalUnsuppressedDigis_cfi.py

```diff
@@ -64,7 +64,10 @@
 
 # remove HE processing for phase 2, completely put in HGCal land
 from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
-phase2_hgcal.toModify(hcalSimBlock, killHE = cms.bool(True) )
+phase2_hgcal.toModify(hcalSimBlock,
+                      doZDCDigi = cms.bool(False),
+                      killHE = cms.bool(True)
+)
 
 # remove HE processing for phase 2, completely put in HGCal land
```

Comment:
Please update also this comment, which was just copy-pasted from the previous one but it does not correspond to what actually implemented here